src/sentry/api/bases/incident.py

```diff
@@ -0,0 +1,11 @@
+from __future__ import absolute_import
+
+from sentry.api.bases.organization import OrganizationPermission
+
+
+class IncidentPermission(OrganizationPermission):
```

Comment:
It looks pretty similar to OrganizationPermission, do we want to have a separate class for some reason?

Reply:
No real reason, I've just seen the convention that we tend to make a new permission class for each object. If that feels wrong I don't mind removing it.

Reply:
Seemed slightly unnecessary to me, but I have no strong feelings about this. I suspect we might have used both conventions in various places.